src/lambda/utils.ts

```diff
@@ -44,11 +44,16 @@ export async function* listLambdaFunctions(client: LambdaClient): AsyncIterableI
 }
 
 /**
- * Converts Lambda handler into a filename by stripping the function name and appending the correct file extension.
+ * Parses Lambda handler into a filename by stripping the function name and appending the correct file extension and a function name
```

Comment:
It is over-specific to say _how_ the function is implemented in the docstring comment (except for cases where that detail is important to the caller to differentiate amongst other similar functions).

Problem with being over-specific is that

- it's a lot of unnecessary detail for someone trying to select a function
- it requires more churn (doc updates or drift) if the implementation changes

Even reading this docstring more than once, it is not clear what its purpose is or why I need all these details. If I care about "parsing" or "stripping the function name and appending" I would read the code (and it would be clearer).

Suggest something like:

```suggestion
 * Gets the filename and function name mapped to a Lambda handler.
```